From d0f85a14d7ab9730f59de040db25bd1ad57755b4 Mon Sep 17 00:00:00 2001
From: Alexandre Torgue <alexandre.torgue@foss.st.com>
Date: Mon, 21 Dec 2020 15:48:05 +0100
Subject: [PATCH 0064/1141] arm64: defconfig: enable STM32 machine support in
 addons fragment

Main flag to build MP2 in 64 bits configuration.

Signed-off-by: Alexandre Torgue <alexandre.torgue@foss.st.com>
Change-Id: I9ebbb4d02bac90836b119ee110bc5f7da4c462b7
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/190002
Reviewed-by: CITOOLS <smet-aci-reviews@lists.codex.cro.st.com>
Reviewed-by: Alexandre TORGUE <alexandre.torgue@st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@st.com>
Tested-by: Alexandre TORGUE <alexandre.torgue@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -1 +1,2 @@
 #  ALPHABETICALLY SORTED: $LC_ALL=C sort fragment-02-defconfig-addons.config -o fragment-02-defconfig-addons.config
+CONFIG_ARCH_STM32=y
